### ISSUE TITLE
ProGuard obfuscation fails during release: unresolved org.xembly.* references

### ISSUE THREAD

**Issue by volodya-lombrozo** [objectionary/lints#1312]
## What happened

The `@rultor release` for tag `0.4.7` failed. Full log: https://www.rultor.com/t/61551-5377748858

Reported in #1295.

## Error

ProGuard obfuscation fails with 56 unresolved references, all pointing to `org.xembly.*` classes (`Directive`, `Directives`, `Xembler`) used by `org.eolang.parser.*` classes (`Emit`, `Eo`, `EoSyntax`, `Listing`, `StrictXmir`):

```
[proguard] Warning: org.eolang.parser.EoSyntax: can't find referenced class org.xembly.Xembler
...
[proguard] Warning: there were 56 unresolved references to classes or interfaces.
[proguard]          You may need to add missing library jars or update their versions.
[proguard] Unexpected error
[proguard] java.io.IOException: Please correct the above warnings first.
```

Build fails at:
```
Failed to execute goal com.github.wvengen:proguard-maven-plugin:2.7.0:proguard (default) on project lints: Obfuscation failed (result=1)
```

## Likely cause

The `xembly` dependency (used transitively via `eo-parser`) isn't declared as a library jar for the `proguard-maven-plugin` configuration in `pom.xml`, so ProGuard can't resolve those classes when analyzing the shaded/obfuscated jar.

## Suggested fix

Either:
- Add `xembly` to the ProGuard plugin's `libs`/library jars configuration, or
- Add a `-dontwarn org.xembly.**` rule if these classes are safe to leave unresolved (e.g. optional/reflection-only usage).

This blocks releases until fixed.

**Comment by volodya-lombrozo** [comment]
@rultor release, tag is `0.4.8`, title is `0.4.8`.

**Comment by volodya-lombrozo** [comment]
@yegor256 The new version was successfully published.
